12c_jdk8/Dockerfile

```diff
@@ -29,15 +27,14 @@ ENV PATH $PATH:$M2_HOME/bin:$JAVA_HOME/bin:$ORACLE_HOME/bin
 # 3. install the jdeveloper in ilent mode
 # 4. cleanup
 
-RUN tar -xvf ./jdk-8u131-linux-x64.tar.gz -C /usr/java && \ 
+RUN tar -xvf ./jdk-8u131-linux-x64.tar.gz -C /usr/java && \
     tar -xvf ./apache-maven-3.5.0-bin.tar.gz -C /usr/apache && \
-	useradd oracle && \
+    useradd oracle && \
     chown -R oracle:oracle /usr/oracle && \
-    su oracle -c "java -jar jdev_suite_122120.jar \
+    su oracle -c "java -jar jdev_suite_122100.jar \
     -silent -responseFile /usr/oracle/middleware/jDevResponseFile \
     -invPtrLoc /usr/oracle/middleware/oraInst.loc" && \
-    rm -rf jdev_suite_122120.jar \
-	jdev_suite_1221202.jar \
+    rm -rf jdev_suite_1221202.jar \
```

Comment:
This rm line does not have the right name, needs to be changed to the right jar.

everything else LGTM